src/main/resources/static/js/post.js

```diff
@@ -5,7 +5,7 @@ const singleCommentTemplate = `
         <div class="media" style="padding-top: 10px;">
             <div class="media-left" style="margin-right: 0px;">
                 <figure class="image is-32x32">
-                    <img class="is-rounded" :src="comment.author.imageUrl" alt="Image">
+                    <img class="is-rounded" :src="${comment.author.imageUrl.startsWith('http') ? comment.author.imageUrl : comment.author.imageUrl.substring(comment.author.imageUrl.indexOf('8080/')+4)}" alt="Image">
```

Comment:
Instead of these modify the code in the backend to set the imageUrl right like this:
```java
        MultipartFile file = post.getContentFile();
        if (file != null) {
            String contentUrl = fileService.saveFile(file, rootDirectory + "post/");
            contentUrl = contentUrl.substring(contentUrl.lastIndexOf("post/"));
            post.setContentUrl("/" + contentUrl);
        }
```

This way you dont need to check this way in the fron-end, only to render ${comment.author.imageUrl}

Reply:
Much better idea